streamer/controller_node.py

```diff
@@ -157,29 +150,55 @@ def start(self, output_dir: str,
     self._input_config = InputConfig(input_config_dict)
     self._pipeline_config = PipelineConfig(pipeline_config_dict)
 
+    if not streamer.util.is_url(output_location):
+      # Check if the directory for outputted Packager files exists, and if it
+      # does, delete it and remake a new one.
+      if os.path.exists(output_location):
+        shutil.rmtree(output_location)
+      os.mkdir(output_location)
+    else :
+      # Check some restrictions and other details on HTTP output.
+      if not self._pipeline_config.segment_per_file:
+        raise RuntimeError(
```

Comment:
We're using RuntimeError in a few similar places, so I think this is okay for now.  I'd be fine with a new error type, if you like, and it may become necessary or useful to have one in the future.  But I'm not worried about it at the moment.